molecule/ssl/verify.yml

```diff
@@ -0,0 +1,22 @@
+- import_playbook: ../common/verify.yml
+
+- name: Check log for successful SSL configuration messages
+  hosts: qdrtest
+  tasks:
+    - name: Fail if we do not find our SslProfileEntity
+      fail:
+        msg: Log does not show correct SslProfileEntity line
+      # "2020-09-15 20:06:48.446910 +0000 AGENT (debug) Add entity: SslProfileEntity(caCertFile=bogus, certFile=bogus, identity=sslProfile/qdrtest-ssl, name=qdrtest-ssl, privateKeyFile=bogus, type=org.apache.qpid.dispatch.sslProfile)",
+      when: qdrlog_content.find('SslProfileEntity(caCertFile=bogus, certFile=bogus, identity=sslProfile/qdrtest-ssl, name=qdrtest-ssl, privateKeyFile=bogus, type=org.apache.qpid.dispatch.sslProfile') == -1
```

Comment:
Both this `.find()` and the `| regex_search()` below were tested to fail if I change the strings to `ZZ-SslProfileEntity` and `ZZ-ListenerEntity` respectfully. I note this because conditionals, filters, and string processing in Ansible can be subtle and they fooled me more than once while I was working on this!